pkg/prommetrics/prommetrics.go

```diff
@@ -120,12 +120,33 @@ var (
 		},
 		[]string{"type", "namespace"},
 	)
+
+	scaledObjectLatency = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: DefaultPromMetricsNamespace,
+			Subsystem: "scaled_object",
+			Name:      "latency",
+			Help:      "ScaledObject Latency",
+		},
+		[]string{"namespace", "scaledObject"},
+	)
+
+	scaledJobLatency = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: DefaultPromMetricsNamespace,
+			Subsystem: "scaled_job",
+			Name:      "latency",
+			Help:      "ScaledJob Latency",
+		},
+		[]string{"namespace", "scaledJob"},
+	)
```

Comment:
I'm not sure if we should expose both metrics or just one metric with both merged, something like `scalableObjectLatency`. 
I don't see use cases where you are interested in monitoring only one of them even though you are using ScaledObjects and ScaledJobs, but I see problems writing promql queries if both aren't available (because they don't have one of them yet)
WDTY @zroubalik ?